src/pandas_profiling/model/pandas/correlations_pandas.py

```diff
@@ -138,3 +151,56 @@ def pandas_phik_compute(
         correlation = phik_matrix(df[selected_cols], interval_cols=list(intcols))
 
     return correlation
+
+
+@Auto.compute.register(Settings, pd.DataFrame, dict)
+def pandas_auto_compute(
+    config: Settings,
+    df: pd.DataFrame,
+    summary: dict,
+    n_bins: int = 10,
```

Comment:
Could we document how the user can change the `n_bins` argument?
(Personally I'd add it as an option under `auto`, just below `threshold` in `config.yml`, with an additional field added to the `Correlation` class in `config.py`)

Does the overload on `Auto.compute` still work when passing `n_bins`?

Reply:
1. I have taken your approach and implemented this as you have recommended. 
2. I have tested calling the pandas_auto_compute function multiple times with/without n_bins parameter without failure. For instance it passes with the following example:

```
auto_result = pandas_auto_compute(test_config, df, summary, n_bins = 12)
auto_result = pandas_auto_compute(test_config, df, summary)
auto_result = pandas_auto_compute(test_config, df, summary, n_bins = 10)

```

Let me know if this is what you mean!